vast/errors.go

```diff
@@ -3,91 +3,96 @@ package vast
 import "errors"
 
 // Please keep the variables alphabetized.
+var (
+	ErrAdSystemMissSystem = errors.New("AdSystem misses system.")
```

Comment:
If you've made this many changes in this file, can you also remove the periods at the end, so that it's more idiomatic?

Otherwise, please revert this file so that the diff reflects the important updates.